BUILD/familiars/gremlins.dat

```diff
@@ -2,5 +2,6 @@
 Nosy Nose
 Plastic Pirate Skull
 Space Jellyfish
+Peace turkey
```

Comment:
```suggestion
Peace Turkey
```
not sure if case sensitive but inconsistent

Reply:
It works as is, but updating for consistency